### PR TITLE
chore: update rector to ^0.13.3

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

File: src/Controllers/LoginController.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function loginAction()
             return redirect()->route('login')->withInput()->with('error', $result->reason());
         }
 
-        $user = $result->extraInfo();
+        $result->extraInfo();
```
Comment (pjsde) [inline src/Controllers/LoginController.php:42]: I think this line can be removed as it is not being used for anything

File: src/Controllers/LoginController.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ public function loginAction()
      */
     public function logoutAction()
     {
-        $user = auth()->user();
+        auth()->user();
```
Comment (pjsde) [inline src/Controllers/LoginController.php:60]: I think this line can be removed as it is not being used for anything

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks good after the removals that @pjsde noted

File: src/Controllers/LoginController.php (hunk 1 above)
Comment (kenjis) [reply src/Controllers/LoginController.php]: Removed.

File: src/Controllers/LoginController.php (hunk 2 above)
Comment (kenjis) [reply src/Controllers/LoginController.php]: Removed.

**Review by MGatner — APPROVED** [review]

**Pull request merged**